pynest/examples/pynest-microcircuit/readme.md

```diff
@@ -0,0 +1,29 @@
+# Microcircuit Readme
+
+Authors: Hendrik Rothe, Hannah Bos, Sacha van Albada   
+May 2016
+
+## Description ##
+This is a PyNEST implementation of the SLI cortical microcircuit from the microcircuit model by Potjans and Diesmann (2014): The cell-type specific
+cortical microcircuit: relating structure and activity in a full-scale spiking
+network model. Cerebral Cortex: doi:10.1093/cercor/bhs358  
+
+* Files:
+	* network.py  
+	file which gathers all parameters and connects the different nodes with each other
+	* network\_params.py  
+	contains the parameters for the network
+	* sim\_params.py  
+	contains the simulation parameters
+	* stimulus\_params.py  
+	contains the parameters for the stimuli
+	* example.py  
+   use this script to try out the microcircuit
+   
```

Comment:
- Please put a '.' behind each sentence.
- Please start the lase sentence with a capital letter: "Use this script..."